xtas/tasks/single.py

```diff
@@ -30,7 +30,7 @@
 
 from cytoolz import identity, pipe
 import nltk
-import spotlight
```

Comment:
Ah, okay. I thought it was a scoping issue, but you're not dragging anything into another namespace here.